src/NuGet.Clients/NuGet.VisualStudio.Implementation/Extensibility/VsPackageInstaller.cs

```diff
@@ -95,7 +95,7 @@ private void RunJTFWithCorrectContext(Project project, Func<Task> asyncTask)
             bool includePrerelease,
             bool ignoreDependencies)
         {
-            RunJTFWithCorrectContext(project, () => InstallPackageAsync(
+            RunJTFWithCorrectContext(project, async () => await InstallPackageAsync(
```

Comment:
I think you should revert all these `async+await` edits since we're in escrow. They add a state machine, another continuation, and while they _should_ be safe, adding them never fixed a deadlock problem AFAIK and theoretically can't really solve anything (aside from casting `Task<T>` to `Task<T2>`).